app/2.0/docs/devguide/data-binding.md

```diff
@@ -697,6 +697,44 @@ to listen for `property-changed` events.  The following constructions are equiva
 <my-element value="{{hostValue}}">
 ```
 
+## Strict binding parser
+
+The default implementation of the binding parser uses a regular expression, due to its better
```

Comment:
Thanks for this! I'd like to suggest alternate text to focus more on the use cases (when you need it, what to do) and less on the implementation. Here's my first take:

===
The default binding parser emphasizes performance over completeness. Some bindings can't be parsed correctly by the default parser—for example, bindings with JavaScript identifiers that include non-ASCII characters.

For a more exact binding parser, use the `StrictBindingParser` mixin. This mixin provides an alternate binding parser that handles all possible cases, although it's slightly less performant than the default parser.
===

I don't love the phrase, "handles all possible cases," but I can't think of anything better at the moment, either.

Reply:
Thanks for the feedback. Regarding the last sentence, what about something like this:
This mixin provides an alternate binding parser that supports Unicode characters, although it's slightly less performant than the default parser.